explainer.md

```diff
@@ -1,751 +1,423 @@
-  negative z-index elements are ordered before positive z-index elements.
+The above is also captured for the incoming state, although the image of the element is 'live', as in it's connected to the representation currently in the DOM.
 
-### How are transition elements painted?
+It's valid for some transition elements to only exist on one side of the DOM change.
 
-During the transition a new stacking context (called uber-root) is created with
-the following hierarchy :
+## Building the pseudo-element tree
+
+For each transitioning element (including the root), the following pseudo-element tree is created:
 
 ```
-uber-root stacking context
-├─ root stacking context
-└─ transition stacking context
+::page-transition-container(name)
+└─ ::page-transition-image-wrapper(name)
+   ├─ ::page-transition-outgoing-image(name)
+   └─ ::page-transition-incoming-image(name)
 ```
 
-This allows using the output of the root stacking context to provide a live root
-image for Page-B. The transition stacking context maps to the transition-root
-element. An alternate approach to this was to paint the transition-root in the
-[top layer](https://fullscreen.spec.whatwg.org/#top-layer). But that made it
-difficult to support transitions when there is other content in the top layer
-(fullscreen elements, dialog) and to ensure effects on the root element which
-are applied to the root stacking context (background-color, filter) are captured
-in the root image. See
-[issue](https://github.com/WICG/shared-element-transitions/issues/74) for
-detailed discussion.
+Where `name` is the `page-transition-tag` value, which is `root` for the root. If the transitioning element only existed on one side of the DOM change, then either the outgoing or incoming image will be missing.
+
+These trees are inserted into a `::page-transition` pseudo element, according to the render-order of their transition elements.
+
+The `::page-transition` is rendered in a top-level, filling the viewport.
+
+Once this is in place, rendering is resumed.
+
+# Default styles & animation
+
+## `::page-transition-container(*)`
 
-## Part 3: The transition
+Default styles:
 
-Everything is now in place to perform the transition. The developer can animate
-the transition elements created by the UA using the usual APIs, such as CSS and
-web animations.
+- Absolutely positioned to the top left of the parent.
+- Width and height of the outgoing element.
+- A transform that places it in the viewport position of the outgoing element.
 
-https://user-images.githubusercontent.com/93594/141100217-ba1fa157-cd79-4a9d-b3b4-67484d3c7dbf.mp4
+Default animation:
 
-Note that the browser defers displaying elements from Page-B and starting the
-animation until Page-B is ready for first render. This is currently driven by
-internal browser heuristics and is being standardized in the proposal
-[here](https://github.com/whatwg/html/issues/7131).
+- Width and height animates to the dimensions of the incoming element.
```

Comment:
Here too, the precise CSS animation being created by the browser makes it clearer. One of the confusion that Tab had during the spec review was whether we were using WebAnimations for the default animations.

Reply:
We can chat about this later, as it's not 100% clear to me